pymove/utils/_winmock.py

```diff
@@ -0,0 +1,15 @@
+class WindowsUser:
+  "Mock class for module errors on windows"
+  def __init__(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)